### ISSUE TITLE
Chabot Accessibility Issue- Screen reader is reading the activity every time  from the beginning on every user action

### ISSUE THREAD

**Issue by nibedita10** [microsoft/BotFramework-WebChat#4277]
# Please view our [Technical Support Guide](https://github.com/microsoft/BotFramework-WebChat/tree/main/docs/TECHNICAL_SUPPORT_GUIDE.md) before filing a new issue.

## Version
botchat-es5.js latest
MSAL :authentication
BotChat.App({
						botConnection: getBotConnectionDetail(connection),
						user: user,
						bot: bot,
						resize: 'detect'
					},

Also hosted in Ms teams

## Describe the bug

Tools Used : Dragon, JAWS (recommended by Organization) and Narrator
The tool is reading everything from the beginning of every user activity instead of reading  bot prompts, user is loosing the context.

Text prompt is not all recognized  and read by the screen  reader tool.

Sometimes it doesn't read for the adaptive card and hero card   prompt as well.

Dragon Tool:
In this tool when user says " send/enter" command twice it is sending the message as well as type send text in the text area of chatbot

Ms teams channel  screen reader is unable to read even the herocard prompts.

## Steps to reproduce

1. Install Jaws /narrator tool and run
2.  open chatbot in webchat
3. it will start reading and you can see the behavior

## Expected behavior

It should read properly with sequence so that it can help visually impaired/locomotor  people .

Sometimes these screen reader tools start reading in different language also.

[Bug]

**Comment by ramfattah** [comment]
investigating

**Comment by ramfattah** [comment]
@nibedita10 

Were you able to reproduce this screen reader behavior issue from any of the [Bot Framework Samples](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples)?

**Comment by nibedita10** [comment]
> @nibedita10
> 
> Were you able to reproduce this screen reader behavior issue from any of the [Bot Framework Samples](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples)?

Please use JAWS tool as well and  use Webchannel rather than emulator ..If need we can connect  I can show you the behavior. For some reason If i capture in the video the sound is not coming.

**Comment by ramfattah** [comment]
closing due to inactivity. @nibedita10, please re-open if needed.
